### ISSUE TITLE
ActiveForm support for HTML 5 input Model::rules()

### ISSUE THREAD

**Issue by thejahweh** [yiisoft/yii2#9348]
Would it be possible, to add HTML 5 input rules from the `Model::rules()`?

This would handle a rule like:

``` php
['fieldxyz', 'integer', 'min' => 10, 'max' => 20]
```

as 

``` php
<input type="number" name="fieldxyz" min="10" max="20">
```

**Comment by cebe** [comment]
we already have this for the maxlength attribute so it should be possible to add this.

**Comment by YaroslavShapoval** [comment]
Should these html5-input-rules be automatically generated with Gii?

I mean that we can do something like this:

```php
<?= $form->field($model, 'fieldxyz')->input('number', ['min' => 10, 'max' => 20]) ?>
```

But we should do this manually. And we can make so that this field will be generated automatically from this rule:

```php
[['fieldxyz'], 'integer', 'min' => 10, 'max' => 20]
```

**Comment by samdark** [comment]
Won't be implemented in 2.0.
